databox-test

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m' 
+NC='\033[0m'
+
+function fail {
+    echo -e "[${RED}FAILED${NC}] ${1} \nERROR: ${2}"
+    #output debug information
+    echo "docker version:"
+    docker version
+    echo "docker ps \n"
+    docker ps
+    echo "docker image ls \n"
+    docker image ls 
+    echo "docker service ls \n"
+    docker service ls
+    echo "service logs databox_container-manager \n"
+    docker service logs databox_container-manager
+    echo "\n"
+    exit 1
+}
+
+function success {
+    echo -e "[${GREEN}OK${NC}] $1"
+}
+
+function test_assert {
+  if [ "$1" != "$2" ]
+  then
+    fail "$3" "$1"
+  else
+    success "$3"
+  fi
+}
+
+#Start databox
+export DATABOX_TESTING=1 
+source ./databox-start dev
+
+echo "Sleeping...."
+sleep 60
+docker ps
+
+#can we see the CM UI
+STATUS=$(curl -sL -w "%{http_code}\\n" "http://127.0.0.1:8989/" -o /dev/null)
```

Comment:
odd use of `-o /dev/null`? i would usually just redirect to `/dev/null` and not use the `-o` flag. is there a reason?

Reply:
because 2>&1 /dev/null does not work.

Reply:
Ah. `-w` causes display to `stdout`. Which helpfully causes it to intermingle with the fetched content. Ok. Less of a hack than piping through `tail` I guess.